sdk/nodejs/x/automation/localWorkspace.ts

```diff
@@ -647,3 +665,19 @@ function defaultProject(projectName: string) {
     const settings: ProjectSettings = { name: projectName, runtime: "nodejs" };
     return settings;
 }
+
+export function checkVersionIsValid(minVersion: semver.SemVer, currentVersion: semver.SemVer) {
+    const err = new Error(`Minimum version requirement failed. The minimum CLI version requirement is ${minimumVersion.toString()}, your current CLI version is ${currentVersion.toString()}. Please update the Pulumi CLI.`);
+    if (minVersion.major !== currentVersion.major) {
+        throw err;
```

Comment:
nit: the error message won't make much sense in this case (major version mismatch, vs what is stated in the error).